openapi3/schema_test.go

```diff
@@ -727,3 +727,125 @@ var schemaErrorExamples = []schemaErrorExample{
 		Want: "NEST",
 	},
 }
+
+func TestSchemaDefaultValues(t *testing.T) {
+	for _, example := range defaultExamples {
+		t.Run(example.Title, testDefaultValue(t, example))
+	}
+}
+
+func testDefaultValue(t *testing.T, example schemaDefaultExample) func(*testing.T) {
+	return func(t *testing.T) {
+		baseSchema := example.Schema
+		for ndx, arr := range [][]fmtDflt{example.AllValid, example.AllInvalid} {
+			for _, typ := range arr {
+				schema := baseSchema.WithDefault(typ.dflt)
+				s, f := typ.fmt, schema.WithFormat
+				if typ.pat != "" {
+					s, f = typ.pat, schema.WithPattern
+				}
+				schema = f(s)
+
+				err := schema.Validate(context.TODO())
```

Comment:
Since this is a test maybe use a short timeout context? Just to fail fast in unit tests.

Reply:
this context is never used in the code, but is in the signature, prehaps in preparation for future work... However, this is a good suggestion to push forward, so am changing it...